setup.py

```diff
@@ -24,7 +24,7 @@
 
 # What packages are required for this module to be executed?
 REQUIRED = [
-    # 'requests', 'maya', 'records',
+    'numpy'  # 'requests', 'maya', 'records',
```

Comment:
I removed it, but I think all packages that are required to run use this package should be listed there

Reply:
For most packages I agree.
But scientific packages are a bit special and often they should be installed with conda.
The problem is that pip does not show, who installed a package and this can be confusing.